arc-theme-upgrade

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Script to upgrade/install the arc-theme
+
+# Remove previows files
+rm -rf /tmp/arc-theme
+
+git clone https://github.com/horst3180/arc-theme /tmp/arc-theme --depth 1 && cd /tmp/arc-theme
+
+# Remove old versions
+sudo rm -rf /usr/share/themes/{Arc,Arc-Darker,Arc-Dark}
+rm -rf ~/.local/share/themes/{Arc,Arc-Darker,Arc-Dark}
+rm -rf ~/.themes/{Arc,Arc-Darker,Arc-Dark}
+
+GNOME_VERSION=$(apt-cache show gnome-shell | grep Version | cut -d ' ' -f2 | cut -c1-4)
+./autogen.sh --prefix=/usr --with-gnome=${GNOME_VERSION}
```

Comment:
Ok, I am with you, but I reported a related bug on issue [#285](https://github.com/horst3180/Arc-theme/issues/285) and you answer me [this](https://github.com/horst3180/Arc-theme/issues/285#issuecomment-150946785), then I fixed with this ```--with-gnome```.  I will try again if I still have problems with that.